src/OmniSharp.Roslyn/Options/IWorkspaceOptionsProvider.cs

```diff
@@ -5,6 +5,8 @@ namespace OmniSharp.Roslyn.Options
 {
     public interface IWorkspaceOptionsProvider
     {
-        OptionSet Process(OptionSet currentOptionSet, OmniSharpOptions omniSharpOptions);
+        OptionSet Process(OptionSet currentOptionSet, OmniSharpOptions omniSharpOptions, IOmniSharpEnvironment omnisharpEnvironment);
+
+        int Order { get; }
```

Comment:
Public api change... but I guess we don't publish packages yet... so we're good.

Reply:
indeed. this will be one of the key extensibility points of OmniSharp so we have to get it right, hopefully now it's good, it's changed 3 times already to facilitate various features 😀we do have 3 feature implementations on top of it already too